### PR TITLE
Added: Landing page and navbar

### PR DESCRIPTION
**Pull request by muskaanshah**
image
image
image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,41 +1,15 @@
...
-            rel="noreferrer"
-          >
-            Contribute
-          </a>
-        </div>
-      </header>
+      <Navbar />
+      <Routes>
+        <Route path="/" element={<Home />} />
+        {/* <Route path="about" element={<About />} /> */}
```
Comment (shobit1337) [inline src/App.js:11]: Avoid pushing commented code

File: src/components/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+			<ul className="nav bg-primary px-1">
+				<div className="section1">
+					<div className="hamburger-menu">
+						<button className="btn-hamburger-menu">
+							<span className="material-icons-outlined"> menu </span>
+						</button>
+					</div>
+					<li className="nav-logo">
+					<Link to="/">
+							<img className="nav-logo-img" src="https://res.cloudinary.com/ecommerce-muskaan/image/upload/v1647541606/e-commerce/dice-logo_sbyevn.png" />
```
Comment (shobit1337) [inline src/components/Navbar/Navbar.jsx]: img tag should always have an alt attribute

File: src/pages/Home/Home.jsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+						src="https://res.cloudinary.com/ecommerce-muskaan/image/upload/v1647541607/e-commerce/landingpage_gobzt1.jpg"
+						className="img-responsive"
+						alt="landingpage"
+					/>
+				</picture>
+			</div>
+			<h1>Categories</h1>
+			<div className="categories mx-auto">
+				{CATEGORIES.map(category => (
+					<GameCard category={category} />
```
Comment (shobit1337) [inline src/pages/Home/Home.jsx:24]: whenever rendering elements using through a map, Always provide it with a key attribute to uniquely identify them.

File: src/pages/Home/components/Gamecard.jsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+function GameCard({category: {imgUrl, categoryName}}) {
+	return (
+		<div className="game-card">
+			<img src={imgUrl} className="img-responsive" />
```
Comment (shobit1337) [inline src/pages/Home/components/Gamecard.jsx]: provide alt attribute to img tag

**Review by shobit1337 — APPROVED** [review]
Suggested some changes, Rest looks good!

File: src/components/Navbar/Navbar.jsx [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+							<span className="material-icons-outlined"> menu </span>
+						</button>
+					</div>
+					<li className="nav-logo">
+					<Link to="/">
+							<img className="nav-logo-img" alt="logo" src="https://res.cloudinary.com/ecommerce-muskaan/image/upload/v1647541606/e-commerce/dice-logo_sbyevn.png" />
+					</Link>
+					</li>
+					<div className="drawer" data-visible="false">
+						<li className="nav-item">
```
Comment (xsaurabhsuthar) [inline src/components/Navbar/Navbar.jsx:20]: Suggestion: You can extract it to a nav component.

**Review by xsaurabhsuthar — APPROVED** [review]
Good to merge 🚀.

**Pull request merged**